samples/net/mqtt_publisher/overlay-bt.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_BT=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
+CONFIG_BT_DEVICE_NAME="Zephyr MQTT"
+CONFIG_NET_L2_BT=y
+CONFIG_NET_IPV4=n
```

Comment:
Setting the follwing:
```
CONFIG_NET_CONFIG_MY_IPV4_ADDR=""
CONFIG_NET_CONFIG_PEER_IPV4_ADDR=""
```
will prevent cmake from throwing warnings during project generation.

Reply:
For BL/IPSP code, we could also set CONFIG_NET_CONFIG_NEED_IPV4=n as IPv4 is not really needed there.